ext/openssl/tests/sni_server_key_cert.phpt

```diff
@@ -0,0 +1,70 @@
+--TEST--
+sni_server with separate pk and cert
+--SKIPIF--
+<?php
+if (!extension_loaded("openssl")) die("skip openssl not loaded");
+if (OPENSSL_VERSION_NUMBER >= 0x10100000) die("skip OpenSSL < v1.1.0 required");
```

Comment:
It's an old sni test that is not valid CA in OpenSSL 1.1 (see the last note in the description). I plan to regenerate and clean the certs up soon so there is actually no point to do that now. Also I want to back port that clean up so we have working sni_server.phpt on OpenSSL 1.1.